azure-pipelines.yml

```diff
@@ -0,0 +1,125 @@
+trigger:
+- master
+
+pr:
+- master
+
+jobs:
+- job: Windows
+  strategy:
+    maxParallel: 1
+  pool:
+    vmImage: 'vs2017-win2016'
+  variables:
+    # Set au version to use or omit to use the latest. Specify branch name to use development version from Github
+    au_version:
```

Comment:
Should it be specified because of 'wormies-au-helpers' = '0.3.2'?

Reply:
This uses latest released version which should be fine (was changed from `master` in #60)